extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -304,6 +304,20 @@ public function getWikiVariables() {
 	}
 
 	/**
+	 * @desc Returns redirected article or null
+	 *
+	 * @param $title
+	 * @return mixed
+	 */
+	public function getRedirectedTitleFromTitle($title) {
```

Comment:
if it's public please cache it.
If it doesn't have to be public please change it to private.

All public methods in Controllers are accessible via API and this method is going straight to DB.